build_scripts/setup_directories.sh

```diff
@@ -1,6 +1,3 @@
-mkdir redi2
-pushd redi2
-virtualenv -p python3 venv
-source venv/bin/activate
-mkdir repos
-popd
+mkdir -p redi2/repos
+mkdir -p redi2/packages
```

Comment:
since it always makes the packages directory, will there only be one path to installation now instead of the multiple there were before?

If that is the case or if the process has changed any please adjust the main README